pinot-core/src/main/java/org/apache/pinot/core/operator/docvalsets/ProjectionBlockValSet.java

```diff
@@ -50,6 +55,30 @@ public ProjectionBlockValSet(DataBlockCache dataBlockCache, String column, DataS
     _dataBlockCache = dataBlockCache;
     _column = column;
     _dataSource = dataSource;
+    _nullBitmapSet = false;
```

Comment:
_nullBitmap is null by default and after initialization it can be either null or non-null. _nullBitmapSet is there to differentiate between _nullBitmap is null and initialized vs. not-initialized.

Reply:
What I meant is that assigning `false` to a member boolean in constructor is redundant, but this is a nitpicking so can be ignored

Reply:
fixed